apps/meteor/client/components/Omnichannel/normalizeDepartments.ts

```diff
@@ -0,0 +1,20 @@
+import type { EndpointFunction } from '@rocket.chat/ui-contexts';
+
+import type { DepartmentListItem } from './hooks/useDepartmentsList';
+
+export const normalizeDepartments = async (
+	departments: DepartmentListItem[],
+	selectedDepartment: string,
+	getDepartment: EndpointFunction<'GET', '/v1/livechat/department/:_id'>,
+) => {
```

Comment:
I'd recommend adding a return type in this case just to ensure the item is being formatted correctly.